manticore/native/cpu/arm.py

```diff
@@ -455,12 +464,7 @@ def _shift(cpu, value, _type, amount, carry):
             amount = 1
```

Comment:
```
# XXX: Capstone should set the value of an RRX shift to 1, which is
# asserted in the manual, but it sets it to 0, so we have to check
if _type in (cs.arm.ARM_SFT_RRX, cs.arm.ARM_SFT_RRX_REG) and amount != 1:
```
While we are here, could you check if this is still the  case in latest Capstone (4.0.1), please?